de.seronet_projekt.mixedport.ros.feature/feature.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature
+      id="de.seronet_projekt.mixedport.ros"
+      label="ROS-MixedPort"
+      version="1.0.0.qualifier"
+      provider-name="SeRoNet (www.seronet-projekt.de)">
+
+   <description url="http://www.seronet-projekt.de/">
+      This feature extends the SmartMDSD Toolchain by a ROS MixedPort
+Component functionality that has been developed by the SeRoNet Project consortium: 
+http://www.seronet-projekt.de
+The ROS Mixed-Port Component is developed by:
+- Service Robotics Research Center of Ulm University of Applied
+Sciences
+- Fraunhofer Institute for Manufacturing Engineering and Automation
+IPA
+CAUTION: 
+This software is a preview for the purpose of technology demonstration.
+It is experimental and comes with no support. Use at your own
+risk.
+   </description>
+
+   <copyright url="http://www.seronet-projekt.de/">
+      Copyright (c) 2018/2019, Servicerobotics Ulm and Fraunhofer IPA
+All rights reserved.
+   </copyright>
+
+   <license url="about.html">
+      Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+* Redistributions of source code must retain the above copyright notice, this
+  list of conditions and the following disclaimer.
+
+* Redistributions in binary form must reproduce the above copyright notice,
+  this list of conditions and the following disclaimer in the documentation
+  and/or other materials provided with the distribution.
+
+* Neither the name of the copyright holder nor the names of its
+  contributors may be used to endorse or promote products derived from
+  this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS &quot;AS IS&quot;
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+   </license>
+
+   <requires>
+      <import feature="org.smartmdsd.toolchain"/>
```

Comment:
I am getting here a warning that says: "cannot not be resolved", however I think this is too generic and the version should be specified.

If I let eclipse to list the required (pressing the button `compute`) imports I get this following list:
```
   <requires>
      <import plugin="org.eclipse.xtext.ide"/>
      <import plugin="org.eclipse.xtext.xbase.ide"/>
      <import plugin="org.antlr.runtime" version="3.2.0" match="perfect"/>
      <import plugin="org.eclipse.xtext"/>
      <import plugin="org.eclipse.xtext.xbase"/>
      <import plugin="org.eclipse.equinox.common" version="3.5.0" match="greaterOrEqual"/>
      <import plugin="org.eclipse.xtext.xbase.lib" version="2.14.0" match="greaterOrEqual"/>
      <import plugin="org.eclipse.xtext.util"/>
      <import plugin="org.eclipse.xtend.lib" version="2.14.0" match="greaterOrEqual"/>
      <import plugin="org.apache.log4j"/>
      <import plugin="org.eclipse.xtext.ui"/>
      <import plugin="org.eclipse.xtext.ui.shared"/>
      <import plugin="org.eclipse.xtext.ui.codetemplates.ui"/>
      <import plugin="org.eclipse.ui.editors" version="3.5.0" match="greaterOrEqual"/>
      <import plugin="org.eclipse.ui.ide" version="3.5.0" match="greaterOrEqual"/>
      <import plugin="org.eclipse.ui"/>
      <import plugin="org.eclipse.compare"/>
      <import plugin="org.eclipse.xtext.builder"/>
      <import plugin="org.eclipse.core.runtime"/>
      <import plugin="org.eclipse.core.resources"/>
      <import plugin="org.eclipse.xtext.generator"/>
      <import plugin="org.eclipse.emf.mwe.utils"/>
      <import plugin="com.google.inject"/>
      <import plugin="org.eclipse.core.commands"/>
      <import plugin="org.eclipse.ui.workbench"/>
      <import plugin="org.xtend.smartsoft.generator"/>
      <import plugin="org.ecore.component.componentDefinition"/>
      <import plugin="org.xtext.component.componentDefinition"/>
      <import plugin="org.ecore.component.seronetExtension"/>
      <import plugin="org.eclipse.emf.edit"/>
      <import plugin="org.ecore.base.mixedport"/>
      <import plugin="org.ecore.base.mixedport.edit"/>
      <import plugin="org.eclipse.emf.ecore"/>
      <import plugin="de.fraunhofer.ipa.componentInterface"/>
      <import plugin="de.fraunhofer.ipa.componentInterface.xtext.ui"/>
      <import plugin="javax.inject"/>
      <import plugin="de.seronet_projekt.ros.interfaces.pool.xtext"/>
      <import plugin="de.seronet_projekt.ros.interfaces.pool.xtext.ide"/>
   </requires>
```